python_legacy/iceberg/core/base_transaction.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, ops, start):
     def table(self):
         return self.transaction_table
 
+    # NOTE: function name has typo in the word `comitted`. Kept for backwards compatability in legacy python API.
     def check_last_operation_commited(self, operation):
```

Comment:
Wasn't there a second typo? I would include the comment with both, since these sorts of changes in less-strictly typed languages are harder to fix once they get in (assuming that this function is part of a public API, which we can't really control as much in Python).

People will likely pull some code from `python_legacy` while maaking the new API, and definitely in the new python library we don't want the method to be misspelled 🙂

Reply:
> Wasn't there a second typo? I would include the comment with both, s

@kbendick 
What's mean exactly? (Sorry My native language is not English.)

Reply:
Originally, when you updated the typo, I thought there was two cases of it.

If that's true, then I would put the comment above both functions that are misspelled, so that if somebody goes to copy them they'll update both (and not only one).

Reply:
@kbendick Thank you for giving me a reply.
I already fixed all which I know :)